src/apps/applib/lib/utility.c

```diff
@@ -233,3 +233,16 @@ void randomGeo(GeoCoord* g) {
         (((float)rand() / (float)(RAND_MAX)) * 180.0) - 90.0);
     g->lon = H3_EXPORT(degsToRads)((float)rand() / (float)(RAND_MAX)) * 360.0;
 }
+
+/**
+ * Returns the number of non-invalid indexes in the array.
+ */
+int countActualHexagons(H3Index* hexagons, int numHexagons) {
```

Comment:
Would `countValidHexagons` be more accurate?

Reply:
They're not necessarily validated with `h3IsValid` - just checked for not being 0, which I take to mean "null" or "missing".